requirements.txt

```diff
@@ -27,13 +27,13 @@ PyYAML>=6.0.0,<7.0.0
 rich>=13.3.1,<14.0.0
 rouge-score>=0.1.2,<0.2.0
 sentencepiece>=0.2.0,<0.3.0
-tokenizers>=0.15.2,<0.16.0
+tokenizers>=0.19.1,<0.20.0
 toml>=0.10.2,<0.11.0
 # Habana installer has 2.2.0a0+git8964477 with oneMKL
 torch>=2.2.0a0,<3.0.0 ; python_version == '3.10'
 torch>=2.2.1,<3.0.0 ; python_version != '3.10'
 tqdm>=4.66.2,<5.0.0
-transformers>=4.30.0,<=4.38.2
+transformers>=4.41.2,<5.0.0
```

Comment:
Same problem, this version bump also breaks Intel Gaudi support.

Reply:
I have filed https://github.com/huggingface/optimum/issues/1895 with optimum and asked to lift the upper version bound of transformers. This will also allow us to bump the tokenizers version.

Reply:
I got feedback on my feature request. The next versions of optimum and optimum-habana will support `transformers >= 4.40.0, < 4.41.0`. In the mean time, we have to support `>=4.38.2`.